pilot/pkg/model/service.go

```diff
@@ -202,6 +201,18 @@ const (
 	TrafficDirectionOutbound TrafficDirection = "outbound"
 )
 
+// Visibility defines whether a given config or service is exported to local namespace, all namespaces or none
+type Visibility string
+
+const (
+	// VisibilityPrivate implies namespace local config
+	VisibilityPrivate Visibility = "."
+	// VisibilityPublic implies config is visible to all
+	VisibilityPublic Visibility = "*"
+	// VisibilityNone implies config is visible to none
+	VisibilityNone Visibility = "~"
```

Comment:
Seems a little weird, I can hardly imagine when we need this.

Reply:
CloudFoundry folks wanted this.. They want to start off with no config to any sidecar

Reply:
We want a whitelist-only model, where Sidecars explicitly import any service / virtualservice that is needed.

Reply:
Explicit import is provided by Sidecar ( with the ~ ).

Someone mentioned they want Sidecar to import everything, and be controlled on export side. Don't remember who and the details - but I think we agreed it's a valid case.